README.md

```diff
@@ -37,141 +36,132 @@
 
 4. **Prepare the image for release.**
 
+> **WARNING**: This will remove `scoring.conf`. Back it up somewhere if you want to save it! It will also remove the `aeacus` executable and other sensitive files.
+
 ```
 ./aeacus --verbose release
 ```
 
-> WARNING: This will remove `scoring.conf`. Back it up somewhere if you want to save it! It will also remove the `aeacus` executable and other sensitive files.
-
 ## Screenshots
 
-#### Scoring Report:
+### Scoring Report:
 
 ![Scoring Report](./misc/gh/ScoringReport.png)
 
-#### ReadMe:
+### ReadMe:
 
 ![ReadMe](./misc/gh/ReadMe.png)
 
 ## Features
 
 -   Robust yet simple vulnerability scorer
--   Image deployment (cleanup, README, etc)
+-   Image preparation (cleanup, README, etc)
 -   Remote score reporting
 
-> Note: `aeacus` ships with very weak crypto on purpose. You need to implement your own crypto functions. See the [Adding Crypto](/docs/crypto.md) for more information.
+> Note: `aeacus` ships with weak crypto on purpose. You should implement your own crypto functions if you want to make it harder to crack. See [Adding Crypto](/docs/crypto.md) for more information.
+
+## Compiling
```

Comment:
Maybe add a note that talks about how the MAKEFILE is currently setup to only work on linux?

Reply:
Sure thing, added.